lib/lyber_core/robot.rb

```diff
@@ -21,56 +18,41 @@ def perform(druid)
       end
     end
 
-    # Converts a given step to the Robot class name
```

Comment:
It appears that this is used elsewhere: https://github.com/search?q=org%3Asul-dlss+step_to_classname&type=Code

Are you sure can be removed?

Reply:
I'm aware it's used. That's why this is a major version.  I don't think this is the best way to handle this. Those usages could just give the class name instead.  There's no need for that code to be in this gem.